### DISCUSSION TITLE
Error -4058

### DISCUSSION THREAD

**Discussion by acomtools** [raweee/node-red-contrib-whatsapp-link#40] [Q&A]
I have an error -4058 when I try to install whatsapp link in Windows 10 Enterprise 2016 LTSB
Would you please advised?

here is the log:
859 verbose stack Error: An unknown git error occurred
859 verbose stack     at makeError (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\git\lib\make-error.js:28:13)
859 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\git\lib\spawn.js:37:26
859 verbose stack     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
860 verbose cwd C:\Users\oee\.node-red
861 verbose Windows_NT 10.0.14393
862 verbose node v18.14.0
863 verbose npm  v9.3.1
864 error code ENOENT
865 error syscall spawn git
866 error path git
867 error errno -4058
868 error enoent An unknown git error occurred
869 error enoent This is related to npm not being able to find a file.
869 error enoent
870 verbose exit -4058
871 timing npm Completed in 51261ms
872 verbose unfinished npm timer reify 1684465000758
873 verbose unfinished npm timer reify:loadTrees 1684465000775
874 verbose code -4058
875 error A complete log of this run can be found in:
875 error     C:\Users\oee\AppData\Local\npm-cache\_logs\2023-05-19T02_56_40_523Z-debug-0.log

**Comment by robertnugteren** [comment]
@raweee is there any solution for this problem? I would like to use your node but I can't get it installed :(
